COMETwebapp/Extensions/ThingExtensions.cs

```diff
@@ -43,5 +44,134 @@ public static string GetName(this Iteration iteration)
             var engineeringSetup = (EngineeringModelSetup)iteration.IterationSetup.Container;
             return $"{engineeringSetup.Name} - Iteration {iteration.IterationSetup.IterationNumber}";
         }
+
+        /// <summary>
+        /// Gets a collection of used <see cref="ParameterType" /> inside an <see cref="Iteration" />
+        /// </summary>
+        /// <param name="iteration">The <see cref="Iteration" /></param>
+        /// <returns>The collection of used <see cref="ParameterType" /></returns>
+        public static IEnumerable<ParameterType> GetUsedParameterTypes(this Iteration iteration)
```

Comment:
may i propose to use the term `Query` instead of `Get` to distinguish from getters and setters